hypercall/include/c/x64/amd/mv_rdl_entry_t.h

```diff
@@ -0,0 +1,47 @@
+/// @copyright
+/// Copyright (C) 2020 Assured Information Security, Inc.
+///
+/// @copyright
+/// Permission is hereby granted, free of charge, to any person obtaining a copy
+/// of this software and associated documentation files (the "Software"), to deal
+/// in the Software without restriction, including without limitation the rights
+/// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+/// copies of the Software, and to permit persons to whom the Software is
+/// furnished to do so, subject to the following conditions:
+///
+/// @copyright
+/// The above copyright notice and this permission notice shall be included in
+/// all copies or substantial portions of the Software.
+///
+/// @copyright
+/// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+/// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+/// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+/// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+/// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+/// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+/// SOFTWARE.
+
+#ifndef MV_RDL_ENTRY_T_H
+#define MV_RDL_ENTRY_T_H
+
+#include <stdint.h>
+
+/**
+ * @struct mv_rdl_entry_t
+ *
+ * <!-- description -->
+ *   @brief see /include/uapi/linux/kvm.h in Linux for more details.
+ *   @var mv_rdl_entry_t::reg
+ *   Member reg holds An mv_reg_t or MSR index
+ *   @var mv_rdl_entry_t::val
+ *   Member val holds The value read or to be written
+*/
+
+struct mv_rdl_entry_t
```

Comment:
The comments for each field need to be above the field itself. Otherwise Clang Tidy will get mad as the AST doesn't seem to support this type of Doxygen format (and both Clang and Clang Tidy check Doxygen support in addition to Doxygen). This means that this struct should look like

```
/**
 * @struct mv_rdl_entry_t
 *
 * <!-- description -->
 *   @brief see /include/uapi/linux/kvm.h in Linux for more details.
 */
struct mv_rdl_entry_t
{
    /** @brief stores an mv_reg_t or MSR index */
    uint64_t reg;
    /** @brief stores the value read or to be written */
    uint64_t val;
};
```